packages/core/pluggableElementTypes/models/baseTrackConfig.ts

```diff
@@ -32,21 +32,37 @@ export function createBaseTrackConfig(pluginManager: PluginManager) {
         description: 'anything to add about this track',
         defaultValue: {},
       },
-      textSearchIndexingAttributes: {
-        type: 'stringArray',
-        description:
-          'list of which feature attributes to index for text searching',
-        defaultValue: ['Name', 'ID', 'Description'],
-      },
-      textSearchIndexingFeatureTypesToExclude: {
-        type: 'stringArray',
-        description: 'list of feature types to exclude in text search index',
-        defaultValue: ['CDS', 'exon', 'transcript'],
-      },
+      textSearchConf: ConfigurationSchema('TextSearchConf', {
+        indexingAttributes: {
+          type: 'stringArray',
+          description:
+            'list of which feature attributes to index for text searching',
+          defaultValue: ['Name', 'ID', 'Description'],
```

Comment:
IMO description should not be a default attribute to index, currently CLI is set at Name,ID

Reply:
will remove